File: iniciante/2752/2752.cpp
#include <iostream>
#include <iomanip> // Para manipuladores como setw() e setprecision()

int main() {
    std::string str(50, ' ');  // String de 50 espaços
    str = "AMO FAZER EXERCICIO NO URI";

    std::cout << "<" << str << ">\n";
    std::cout << "<" << std::setw(30) << str << ">\n";
    std::cout << "<" << str.substr(0, 20) << ">\n";
    std::cout << "<" << std::left << std::setw(20) << str << ">\n";
    std::cout << "<" << std::left << std::setw(30) << str << ">\n";
    std::cout << "<" << str.substr(0, 30) << ">\n";
    std::cout << "<" << std::right << std::setw(30) << str.substr(0, 20) << ">\n";
    std::cout << "<" << std::left << std::setw(30) << str.substr(0, 20) << ">\n";

    return 0;
}

File: iniciante/2752/2752.rs
fn main() {
    let mut frase = String::with_capacity(50);
    frase.push_str("AMO FAZER EXERCICIO NO URI");

    println!("<{}>", frase);
    println!("<{:>30}>", frase);
    println!("<{:.20}>", frase);
    println!("<{:<20}>", frase);
    println!("<{:<30}>", frase);
    println!("<{:.30}>", frase);
    println!("<{:>30.20}>", frase);
    println!("<{:<30.20}>", frase);
}